### DISCUSSION TITLE
[Question] Does migrate command run migrations only on apps within schema definition ?

### DISCUSSION THREAD

**Discussion by VaZark** [lorinkoz/django-pgschemas#149] [Q&A] [answered]
Django runs migrations on apps in INSTALLED_APPS. Does `runschema` select the apps from config and run migrations or does it just run all migrations and fake the non-associated apps?

**Comment by lorinkoz** [comment, marked as answer]
All migrations are run. The migrations for apps that are not meant for a schema are "registered as applied" but not actually applied.
